src/topology/continuous_function/algebra.lean

```diff
@@ -582,6 +603,23 @@ instance has_scalar' {α : Type*} [topological_space α]
   has_scalar C(α, R) C(α, M) :=
 ⟨λ f g, ⟨λ x, (f x) • (g x), (continuous.smul f.2 g.2)⟩⟩
 
+instance {α : Type*} [topological_space α] [locally_compact_space α]
+  {R : Type*} [semiring R] [topological_space R]
+  {M : Type*} [topological_space M] [add_comm_monoid M]
+  [module R M] [has_continuous_smul R M] :
+  has_continuous_smul R C(α, M) :=
```

Comment:
Can you show the analogous result for `has_continuous_const_smul` too?

Reply:
You'll want to merge master first.

Reply:
Done